integration-tests/emulated/src/lib.rs

```diff
@@ -19,5 +19,6 @@
 mod mock;
 mod utils;
 
+#[cfg(not(feature = "runtime-benchmarks"))]
```

Comment:
You can combine the two conditions. 
```suggestion
#[cfg(all(not(feature = "runtime-benchmarks"), test))]
```

Reply:
Ah yes. Thank you! [1cda1a4](https://github.com/KILTprotocol/kilt-node/pull/655/commits/1cda1a4925a16eeefa8e8284d0ae1a2efeb3a177)